djangosaml2idp/models.py

```diff
@@ -16,6 +16,7 @@
 from saml2 import xmldsig
 
 from .idp import IDP
+from .settings import SERVICE_PROVIDER_MODEL
```

Comment:
Shouldn't this be `from django.conf import settings`? that way you're actually importing the settings proxy object.

Reply:
Nevermind. I looked back over the Oauth toolkit implementation and it looks like you're doing it the same way.